src/ext/local-vars.js

```diff
@@ -0,0 +1,36 @@
+htmx.defineExtension('local-vars', {
+    onEvent: function (name, evt) {
+        if (!localStorage) {
+            console.log("Warning: browser does not support local storage.")
+        }
+        elt = evt.detail.elt;
+        if (name === "configRequest.htmx") {
```

Comment:
can you update to the new event naming standard: "htmx:configRequest"

Reply:
ok